src/components/Button/Button.js

```diff
@@ -39,14 +44,23 @@ const defaultProps = {
   title: undefined,
   tabIndex: 0,
   style: undefined,
+  accessKey: '',
```

Comment:
undefined by default so the attribute is not added when empty
```suggestion
  accessKey: undefined,
```